core-services/prow/02_config/_plugins.yaml

```diff
@@ -1008,6 +1009,21 @@ external_plugins:
     - issue_comment
     - pull_request
     name: needs-rebase
+  integr8ly/ansible-tower-configuration:
+  - endpoint: http://refresh
+    events:
+    - issue_comment
```

Comment:
Any reason of excluding `- pull_request`?

Reply:
Hi @hongkailiu, I've copied this block from other integr8ly repos (see below). I'm not sure myself why it would be excluded. Should we add it in to them all?

Reply:
My bad. I thought it was the need-rebase plugin.